Sources/Localization/aa.lproj/Localizable.strings

```diff
@@ -184,6 +184,7 @@
 "product-detail.nutrition-table.serving" = "serving";
 
 // Nutrition levels
+"nutrition-level.title" = "Nutrient levels per 100 g/100mL";
```

Comment:
You should fill the translations files only for the langage you add. If you add only the key in english its fine, translation will be done afterwards. Duplicating the english 

1. version in every langage poses several problems: the PR is difficult to review (too many files changed)
2. translation will probably not be done because the system sued will think that the keys are translated

Could you please remove all of them except in english ? thanks

Reply:
👍 good catch